grails-app/controllers/org/bbop/apollo/IOServiceController.groovy

```diff
@@ -93,7 +94,7 @@ class IOServiceController extends AbstractApolloController {
                         }
                     }
                 }
-                'in'('class',Gene.class.name)
+                'in'('class',requestHandlingService.viewableAnnotationList+[Deletion.class.name,Insertion.class.name,Substitution.class.name])
```

Comment:
This is fine for now . . probably good to have somewhere: viewableAlterationList . . . . but there are other refactoring issues with this, so good for now.